### PR TITLE
Run Pipeline on Merge to V2 + Deploy to Dev. Env. Stub

### PR DESCRIPTION
**Pull request by SharonHart**
Decided to have a single pipeline, only have the deployment task under a branch condition:
https://docs.microsoft.com/en-us/azure/devops/pipelines/process/conditions?view=azure-devops&tabs=yaml#run-for-the-main-branch-if-succeeding
In another solution, we will have to duplicate the pipeline and extract its content to a template, making it less readable.

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -1,11 +1,6 @@
 trigger:
   - V2
 
-pr:
-  branches:
-    include:
-      - V2
-
```
Comment (SharonHart) [inline azure-pipelines.yml:8]: Defined in the UI

**Comment by SharonHart** [comment]
Closing, will be done once we add deployment to dev. env.

**Review by navalev — APPROVED** [review]

**Review by omri374 — APPROVED** [review]

**Comment by omri374** [comment]
@SharonHart should this be merged or only after we replace the stub?

**Comment by SharonHart** [comment]
@omri374 Not really, but I see that Avishay's PR is going to be merged soon so this might not have that much content after all. Let's just leave it open for now

**Pull request closed without merging**